roles/foreman_devel/tasks/main.yml

```diff
@@ -1,3 +1,27 @@
 ---
 - include_tasks: github_push_ssh.yml
   when: foreman_devel_github_push_ssh
+
+- include_tasks: check_forks_of_repos.yml
+  when: katello_devel_github_username is defined and foreman_installer_options_internal_use_only is defined
+
+- name: "Check situation, if the machine is behind NAT"
+  # for example OpenStack instances
+  # This is potencional problem, if user has the same private IP in own lan. 
+  shell: "ping -c 1 {{ hostvars[inventory_hostname]['ansible_default_ipv4']['address'] }}"
+  delegate_to: localhost
+  ignore_errors: True
+  register: rping
+
+- name: "Add OS hostname to hosts file" 
```

Comment:
This looks a lot like our ```etc_hosts``` role

Reply:
I think this could just be done, without the check starting at line 8. The `lineinfile` module is meant to only make changes if a file is not present. So seems like there would be no harm in always ensuring there is an entry in `/etc/hosts` ?

Reply:
@ehelms No problem, I'm removing it.

Reply:
done